src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from "react";
+
+import { BaseBadgeProps } from "./type";
+import { StyledBadge } from "./Style";
+import { ReactComponent as IconIcBlack } from "../../assets/icon-ic-black.svg";
+import { ReactComponent as IconIcBlue } from "../../assets/icon-ic-blue.svg";
+import { ReactComponent as IconIcGreen } from "../../assets/icon-ic-green.svg";
+import { ReactComponent as IconIcOrange } from "../../assets/icon-ic-orange.svg";
+import { ReactComponent as IconIcRed } from "../../assets/icon-ic-red.svg";
+import { ReactComponent as IconIcWhite } from "../../assets/icon-ic-white.svg";
+
+const Badge: FC<BaseBadgeProps> = (props) => {
+  const {
+    variant = "light",
+    inverted = false,
+    suffix = false,
+    prefix,
+    iconOnly = false,
+    label = "",
+    children,
+  } = props;
+
+  return (
+    <StyledBadge variant={variant} inverted={inverted}>
+      <div className="badgeWrapper">
+        {!iconOnly ? (
+          <>
+            <div className="prefixBadge">{prefix}</div>
+            <p className="badgeLabel">{label}</p>
+            <>{suffix && renderSuffixIconBadge({ inverted, variant })}</>
+          </>
+        ) : (
+          children
+        )}
+      </div>
+    </StyledBadge>
+  );
+};
+
+export default Badge;
+
+//WA: helpers function
+const renderSuffixIconBadge = ({
+  inverted,
+  variant,
+}: {
+  inverted: boolean;
+  variant: string;
+}) => {
+  return inverted ? (
+    <IconIcWhite />
+  ) : (
+    mapIconBadge[variant as keyof typeof mapIconBadge]
+  );
+};
+
+const mapIconBadge = {
+  light: <IconIcBlack />,
+  neutral: <IconIcBlack />,
+  info: <IconIcBlue />,
+  success: <IconIcGreen />,
+  warning: <IconIcOrange />,
+  critical: <IconIcRed />,
+};
```

Comment:
ini kalo icon badgenya jadi satu aja, terus fillnya aja yang diganti bisa ga ya?

Reply:
kemarin udah pengen gini dan udah minta assets nya ke tim designer tp kata mereka gabisa karena itu satu kesatuan icon. gue coba export sendiri juga export failed
<img width="1440" alt="Screen Shot 2022-07-11 at 13 41 40" src="https://user-images.githubusercontent.com/104057020/178203966-7eb89445-52b7-46ab-826b-c3477031936e.png">

Reply:
ohya dan ini file svgnya if u want to try
![icon-info-transparent](https://user-images.githubusercontent.com/93510602/178221586-4c26e36a-c567-40ce-9752-06dd9dae99de.svg)
![icon-info-white](https://user-images.githubusercontent.com/93510602/178221597-6cad5f2d-e035-461a-9b3f-8a777f833c1c.svg)

Reply:
done ya